recipe/meta.yaml

```
@@ -15,16 +17,21 @@ source:
       - fix_ppc64le.patch  # [ppc64le]
 
 build:
-  number: 1
-  string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+  number: 2
+  string: py{{ PY_VER_MAJOR }}{{ PY_VER_MINOR }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
```

Can you keep the `_{{ build_ext }}`
```suggestion
  string: py{{ PY_VER_MAJOR }}{{ PY_VER_MINOR }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
```
? Otherwise it is impossible to distinguish cuda vs non-cuda builds. This also applies to other build strings in the recipe.

---

Ah yes forgot that add will also add it in the outputs